src/Compilers/VisualBasic/Portable/Symbols/UnboundGenericType.vb

```diff
@@ -338,6 +339,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             Return SpecializedCollections.EmptyEnumerable(Of PropertySymbol)()
         End Function
 
+        Friend Overrides ReadOnly Property AllRequiredMembers As ImmutableSegmentedDictionary(Of String, Symbol)
+            Get
+                Return OriginalDefinition.AllRequiredMembers
```

Comment:
>OriginalDefinition.AllRequiredMembers [](http://example.com/codeflow?start=23&length=37)

It looks like unbound generic types have only types, otherwise those should be substituted. What are we doing for C#?

Reply:
C# doesn't have a dedicate type for unbound generics, it uses ErrorTypeSymbols, which have no members and will calculate an empty set.

Reply:
Are we testing these code paths?